custom_scripts/py_scripts/test_executor/example.py

```diff
@@ -0,0 +1,8 @@
+from typing import Dict, Any
+
+import requests
```

Comment:
You can use functions and classes directly from the module, for example:
```
from requests import head as req_head
response = req_head(config.get("address", "127.0.0.1"))
```
Just in case.